dashboard/lib/policies/ai.rb

```diff
@@ -1,6 +1,15 @@
-require 'user'
-
 class Policies::Ai
+  # Units that enable the Differentiation feature
+  UNIT_DIFFERENTIATION_ALLOWLIST = [
+    'csd3-2023',
+  ]
+
+  # UnitGroups that enable the Differentiation feature
+  UNIT_GROUP_DIFFERENTIATION_ALLOWLIST = [
+    'csd-2023',
+    'csd-2024',
+  ]
```

Comment:
should these other units be included too? https://github.com/code-dot-org/code-dot-org/blob/3409883b211c27e002679ea50ef356a5295c64de/dashboard/app/jobs/concerns/ai_rubric_config.rb#L34-L39

Reply:
I think so! I'll have to make sure.

Reply:
I think it will save us headache in the future to add these now - good call!

Reply:
Easy peasy. Will do.